code/modules/SCP/SCPs/SCP-035.dm

```diff
@@ -0,0 +1,206 @@
+	update_icon()
+	addtimer(CALLBACK(src, .proc/weepstart), rand(9.5, 114) SECONDS) //only 2 min max downtime, it outputs alot
+	if(prob(5))
+		tradegy = !tradegy
+
+/obj/item/clothing/mask/scp_035/proc/output_substance()
+	if(!weeping)
+		return
+	if(prob(80)) //output under the mask 4/5 of the time
+		//new /obj/effect/fluid/scp_035_substance(src.loc)
+		var/obj/effect/decal/cleanable/blood/drip/scp035/drip = new(get_turf(src))
+		drip.basecolor = bleed_colour
+		drip.update_icon()
+	else //find a wall to seep out of
+		var/list/target_list = list()
+		for(var/turf/T in view(5, src))
+			if(T.density && T.opacity)
+				target_list += T
+		new /obj/effect/fluid/scp_035_substance(pick(target_list))
+	addtimer(CALLBACK(src, .proc/output_substance), 10 SECONDS)
+
+/obj/item/clothing/mask/scp_035/equipped(mob/user)
+	. = ..()
+	if(!ishuman(user))
+		return
+	var/mob/living/carbon/human/H = user
+	if(H.wear_mask == src)
+		corrupt_host(H)
+
+/obj/item/clothing/mask/scp_035/proc/corrupt_host(mob/living/carbon/human/H)
+	if(!H)
+		return
+
+	if(H.faction == "SCP")
+		return
+
+	to_chat(H, "<span class='danger'>You feel an ominous presence prodding at the edges of your consciousness...</span>")
+
+	if(H.resist())
+		to_chat(H, "<span class='danger'>You struggle to resist the intrusion, but it pries into your mind relentlessly! You can feel it worming its way through your thoughts, prying open your memories. You try to hold on to who you are, but the sinister will behind this mental attack is too strong!</span>")
+	else if(prob(50))
+		to_chat(H, "<span class='danger'>You attempt to fight back against the mental invasion, but feel your thoughts slowly being overtaken! It's like dozens of icy tendrils plunging into your mind, sifting through your memories and personality. You desperately try to cling to your identity, but it keeps slipping through your grasp!</span>")
+	else
+		to_chat(H, "<span class='danger'>You try to resist the insidious mental attack, but the sinister will behind it overwhelms your own! It feels like your mind is being pried open and rifled through, your sense of self coming undone. Your memories and emotions are laid bare as something ancient and evil takes over.</span>")
+
+	H.Stun(10) // freeze the player for a bit while the takeover happens
+
+	// begin the actual takeover
+	to_chat(H, "<span class='scp035'>Your vision blurs as your mind is flooded with ancient memories that are not your own. They pour into your head, centuries of sinister history and malevolent thoughts.</span>")
+
+	addtimer(CALLBACK(src, .proc/continue_corruption, H), 30 SECONDS)
+
+/obj/item/clothing/mask/scp_035/proc/continue_corruption(mob/living/carbon/human/H)
+	to_chat(H, "<span class='scp035'>You feel your personality unraveling, your memories and identity drained away. It's like you are being hollowed out from the inside, everything that makes you who you are scraped away bit by bit.</span>")
+
+	canremove = FALSE // stop them from taking off the mask
```

Comment:
This should probably be moved much earlier.

Also, allow others to pry the mask off somehow.